modules/oe_theme_content_entity_contact/config/overrides/core.entity_view_display.oe_contact.oe_press.oe_details.yml

```diff
@@ -0,0 +1,83 @@
+langcode: en
```

Comment:
Same as above, we need this changes to be applied in a post update hook.

Reply:
Done. See https://github.com/openeuropa/oe_theme/pull/523/files#diff-c610b1a934c884e2b03e44cd484367a4